### ISSUE TITLE
Current version pushes sidemenu divs over even though they are behind the backdrop shadow..

### ISSUE THREAD

**Issue by WriterStat** [nakupanda/bootstrap3-dialog#166]
When using current downloaded bootstrap-dialog code with Bootstrap 3.3.4. 

The current code can push certain sidemenu divs depending on viewport.  That are behind the backdrop shadow over to fit the dialog in, even though they are behind the backdrop, instead of just hovering over them.

Also on occasion current code dialogs that start out normal sized and should stay normal sized can and do actually super expand to wide size, without cause, depending on changing view port size.

Example I start a dialog normal sized and then make the viewport smaller, the dialog should stay normal sized as I decrease the size of the browser window, then should decrease in size when the browser window meets it or gets smaller than the dialog.  And yet it expands to wide size and fills the entire screen depending on view port size.

These two items above make a site look hinky/bad with things resizing and moving without reason.

Any fix?  Thanks!  Cheers.

**Comment by WriterStat** [comment]
To see the super size in action. Open up the official demo page in firefox and scroll down to the very last one on the demo page, the orange banana text one.  Then slowly decrease the width size of the browser window by dragging it. Once the browser window gets around 768px to 747px in browser view port size.  The normal sized dialog jumps up in width and size to super wide size and fills the entire browser window.  Then from there on decreases.

This is a bit abnormal. =) Hahaha... in my book. =)  Note: this also does this in my tests on other pages.

**Comment by nakupanda** [comment]
Hi,

Sorry for reading this so late, let me investigate.

**Comment by nakupanda** [comment]
Hi,

Does this still need a fix?

**Comment by WriterStat** [comment]
yeah. =) I've been ignoring it. =)  For some reason it only does it if the area the dialog shows up over/above is close or similar in size. 

Example: I have three columns in the web page, two side columns and a middle area basically three divs.

If the middle div is close in width to the dialog.  Then when it opens it pushes the right side column area over to accommodate the dialog's size.  And when it closes the right side column snaps back into it's original place.

However if the middle area div that the dialog shows up over/above is way smaller in width than the dialog, then no movement occurs. Seems only does this when the div under it is close in width to the dialog's width.

I know it sounds crazy. =)))  Possible some collision?

Anyways thanks for everything!  You've been great! Much appreciated, the code and all the cool help.

**Comment by longpham91** [comment]
Yes, I'm seeing this too. The container got "squished" inward as soon as the dialog is open.
